api/v2/serializers/details/provider.py

```diff
@@ -25,12 +25,19 @@ class Meta:
 
 class ProviderSerializer(serializers.HyperlinkedModelSerializer):
     name = serializers.CharField(source='location')
-    type = ProviderTypeSerializer()
-    virtualization = PlatformTypeSerializer()
-    sizes = SizeSummarySerializer(source='size_set', many=True)
     url = UUIDHyperlinkedIdentityField(
         view_name='api:v2:provider-detail',
     )
+    type = ProviderTypeSerializer()
+    virtualization = PlatformTypeSerializer()
+    sizes = SizeSummarySerializer(source='size_set', many=True)
+    is_admin = serializers.SerializerMethodField()
+
+    def get_is_admin(self, provider):
+        user = self.context['request'].user
+        if user.is_staff or user.is_superuser:
+            return True
+        return False
```

Comment:
Mildly-intentional use of this pattern because it allows us to drop in more 'If X: return True" conditionals.. But I have nothing against changing it.